### DISCUSSION TITLE
Own color for scriptlets in the logger

### DISCUSSION THREAD

**Discussion by dimisa-RUAdList** [uBlockOrigin/uBlock-issues#2636] [Ideas] [answered]
Now the scriptlets in the logger have the same color as the cosmetic rules. I think it would be more convenient if the scriptlets had their own color.

**Comment by gorhill** [comment]
https://github.com/gorhill/uBlock/commit/bc0235861046c2980ed9bed008877cd8e577b9ba

**Reply by dimisa-RUAdList** [reply]
@gorhill 
Great! Maybe to complete the idea, add scriptlets here? >
<details>
<summary>Screenshot(s)</summary>

![Logger](https://user-images.githubusercontent.com/20126984/236668048-77de493e-8abf-436b-a493-cc466348e1dc.jpg)

</details>

**Reply by peace2000** [reply]
I have been thinking the same as dimisa. Different category would be very logical choice.

**Reply by gorhill** [reply, marked as answer]
https://github.com/gorhill/uBlock/commit/88fb6b153df088d1b4f4015386498ec15aed53e3
